remote.go

```diff
@@ -45,7 +45,7 @@ type Remote struct {
 	s storage.Storer
 }
 
-func newRemote(s storage.Storer, c *config.RemoteConfig) *Remote {
+func NewRemote(s storage.Storer, c *config.RemoteConfig) *Remote {
```

Comment:
can you document the new exported function?

Reply:
sorry for the late response, I have added the documentation